index.html

```diff
@@ -7,62 +7,64 @@
-					<input type="checkbox" name="dont_contact_me" id="nocontact" />
-					<small>I do not wish to be contacted</small>
+			<label  for="name" class="label h4">Your name (required)</label>
+			<input name="name" id="name" minlength="2" type="text" required>
+			
+			<label for="email" class="label h4">Your email address (required)</label>
+			<input name="email" required type="email" id="email" value="" placeholder="">
+			
+			<label for="what" class="label h4">What happened? (required)</label>
+			<textarea
+				name="what_happened"
+				id="what"
+				required placeholder="Describe what happened, and where it happened. Just a sentence or two is fine!"
+				aria-describedby="Describe what happened, and where it happened. Just a sentence or two is fine!" /></textarea>
+
+			<label for="postcode" class="label h4">Where was it? - Postcode (required)</label>
+			<input
+				name="postcode"
+				minlength="4"
+				maxlength="4"
+				required
+				type="text"
+				id="postcode"
+				value=""
+				placeholder=""
+			>
+
+			<div class="checkbox-container">
+				<input type="checkbox" name="dont_contact_me" id="nocontact" class="checkbox" />
+				<label for="nocontact" class="checkbox-label">
+					I do not wish to be contacted
 				</label>
-			<button type="submit" id="submit_form">Report it.</button>
-			<!--
-			<button class="g-recaptcha" data-sitekey="6LfGRX0UAAAAAINiXRMozu84LiFGNjdKn-QRwgr4" data-callback="onSubmit" type="submit" id="submit_form">Report it.</button> -->
+			</div>
+			<button type="submit" id="submit_form"><h4>Report it</h4></button>
 		</form>
 
-		<p class="submitted">
-			<strong>Thank you!</strong> Your report has been submitted. You will be sent a copy of your report.
-		</p>
-
-		<p class="failed">
-			<strong>There was a problem submitting you form.</strong> Please try again in a few minutes.
-		</p>
-		<footer>
+		<div class="submitted">
+			<h3>Thank you!</h3>
 			<p>
-				<img class="f_left" src="./veohrc_logo_new.png" width="240"/>
-<!-- 				<img class="f_left" src="./veohrc_logo.svg" width="32"/>
-				<small>
-					Victorian Equal Opportunity<br/>
-					<span class="light">& Human Rights Commission</span>
-				</small> -->
-				<!-- insert ICV logo here -->
+				Your report has been submitted. You will be sent a copy of your report.
 			</p>
-		</footer>
-	</main>
-  	<script src="./index.js"></script>
-  	<script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/3.5.3/iframeResizer.contentWindow.min.js"></script>
+		</div>
+
+		<div class="failed">
+			<h3>There was a problem submitting your form.</h3>
+			<p>Please try again in a few minutes.</p>
+		</div>
+	</div>
+	<script src="./index.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/3.5.3/iframeResizer.contentWindow.min.js"></script>
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 	<script src="https://sdk.amazonaws.com/js/aws-sdk-2.283.1.min.js"></script>
```

Comment:
Are you clear on whether this script is used? I don't know if any methods actually get called on the SDK

Reply:
No don't know about this scrip, it was just in the file already. Should I remove it?

Reply:
yeah lets try removing it